spotbugsTestCases/src/java/ghIssues/Issue79.java

```diff
@@ -0,0 +1,30 @@
+package ghIssues;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+public class Issue79 {
+    private static final String QUERY = "";
+
+    public void f(Connection cnx) throws SQLException {
+        PreparedStatement st = null;
+        ResultSet rs = null;
+        try {
+            st = cnx.prepareStatement(QUERY);
+            rs = st.executeQuery();
+            while (rs.next()) {
+                System.out.println(rs.getString("ID"));
```

Comment:
Can this be a proper logging statement?

Reply:
I'd suggest using sl4j loggers instead. println is not fun to deal with when debugging with log statements.

Reply:
I'd say using sl4j loggers instead would be better.

Reply:
@raghav-deepsource 
It seems that this is a test fixture. So, I think we don't need use slf4j loggers here.
But, It looks that we can omit this block.

```suggestion
```

Reply:
@wreulicke, I have modified the code according to your suggestion.  Thanks :)

Reply:
I modified it based on @ThrawnCA's suggestion.
If this block is omitted, SpotBugs will detect `DLS: Dead storage for local variables` bug instance and the test will fail.